src/seedsigner/views/screensaver.py

```diff
@@ -84,7 +84,7 @@ def _render(self):
 
         background = Image.new("RGBA", size=self.logo.size, color="black")
         if not self.is_screenshot_renderer:
-            # Fade in alpha
```

Comment:
We always try to over-explain things via comments. Do not remove unless a comment is wrong, no longer relevant, etc.